pyproject.toml

```diff
@@ -0,0 +1,114 @@
+[build-system]
+requires = ["scikit-build-core"]
+build-backend = "scikit_build_core.build"
+
+[project]
+name = "itk-ants"
+version = "0.19.2"
+description = "Provides an ITK Python interface to elastix, a toolbox for rigid and nonrigid registration of images"
+readme = "README.md"
+license = {file = "LICENSE"}
+authors = [
+    { name = "Matthew M. McCormick", email = "matt.mccormick@kitware.com" },
+    { name = "ViktorvdValk", email = "viktorvandervalk@gmail.com" },
+    { name = "Niels Dekker", email = "N.Dekker@lumc.nl" },
+    { name = "Tom Birdsong" },
+    { name = "Konstantinos Ntatsis", email = "konstantinos.ntatsis323@gmail.com" },
+]
+keywords = [
+    "itk",
+    "elastix",
+    "registration",
+]
+classifiers = [
+    "Development Status :: 4 - Beta",
```

Comment:
Should we bump this up to some more mature level?

Reply:
No, I think we are still beta.